pages/embed/contribution-flow.js

```diff
@@ -149,6 +153,9 @@ class NewContributionFlowPage extends React.Component {
     }
 
     const contributionBLocker = getContributionBlocker(LoggedInUser, account, tier, Boolean(this.props.tierId));
+    if (contributionBLocker.reason === CONTRIBUTION_BLOCKER.NO_CUSTOM_CONTRIBUTION) {
+      return <Redirect to={`${account.slug}/contribute`} />;
+    }
```

Comment:
This will crash if there's no contribution blocker. We should move this inside the `if (contributionBLocker) {`